ui/raidboss/data/05-shb/raid/e6s.js

```diff
@@ -76,14 +76,14 @@
           safeZoneObj1 = {
             en: 'north',
             de: 'nord',
-            fr: 'nord',
+            fr: 'nord ',
```

Comment:
Can you add the space only if necessary, maybe in the `concatSafeZones` function below? Otherwise if the same zone is north, then this will return `nord ` (with a trailing space).

Reply:
Sorry but I'm not good enough in prog. to do that. But if you tell me how to add a space or a "-" between Obj1&Obj2 I'll do it with great pleasure.

Reply:
Sorry for going ahead and just doing this myself, but I did this in ac0987a if you wanted to see how to do it.

Reply:
Np & Thanks a lot for the lesson :)